sdk/core/azure-core-http-jdk-httpclient/src/main/java/com/azure/core/http/jdk/httpclient/JdkHttpClient.java

```diff
@@ -156,7 +158,7 @@ private java.net.http.HttpRequest toJdkHttpRequest(HttpRequest request, Context
      *
      * @return the java major version
      */
-    private int getJavaVersion() {
+    private static int getJavaVersion() {
```

Comment:
Is this method better off in CoreUtils or some helper method in core for other libraries to use?

Reply:
I'd rather leave this to just the JDK HttpClient for now, don't want this used too commonly